Cogs/verification.py

```diff
@@ -18,52 +18,33 @@ class Verification(commands.Cog):
     def __init__(self, bot):
         self.bot=bot
 ###########################################################################
-    @commands.command(name='verify')
-    async def verify(self, ctx, email):
+    @discord.slash_command(name='verify', description="Verifies a user for access to locked channels")
+    async def verify(self, ctx, email, firstname, lastname):
         if(discord.utils.get(ctx.author.roles, name = "Verified") != None): # We could have this check the DB, would maybe cause issues with manually verified folks.
-            await ctx.send(embed = discord.Embed(title = "You are already verified!", description = "*If you believe this is an error, please message a board member*", color = discord.Color.red()))
-            await ctx.message.delete()
+            await ctx.respond(embed = discord.Embed(title = "You are already verified!", description = "*If you believe this is an error, please message a board member*", color = discord.Color.red()))
             return
-
-        #Check if email has starting/ending parenthesis
-        if (email[0] == "("):
-            email = email[1:]
-        if (email[-1] == ")"):
-            email = email[:-1]
+        
+        #Trims parameters
+        email = email.strip()
+        firstname = firstname.strip()
+        lastname = lastname.strip()
 
         verified, message, color = verify_user(ctx.author.id, email)
 
-        def check(message): #Makes sure user replying equal to user who started the command
-            return message.author.id != self.bot.user.id and message.author.id == ctx.author.id
-
         if verified: #Prompts user to enter name to finish verifying
-            author = ctx.message.author
-            description = f"{ctx.author}, please enter your first and last name to finish the verification process."
-            nameMsg = await ctx.send(embed = discord.Embed(title = message, description = description, color = discord.Color.teal()))
+            author = ctx.author

```

Comment:
probably dont need to save this anymore right? cna just do ctx.author everywhere